Automattic/wp-calypso
# Update/information box typography

Fixes #953

After:
Placeholder 
![screen shot 2015-12-15 at 15 30 00](https://cloud.githubusercontent.com/assets/115071/11827402/e1f94a8c-a340-11e5-91ae-9345ace295e9.png)

Multi site view.
![screen shot 2015-12-15 at 16 12 27](https://cloud.githubusercontent.com/assets/115071/11828153/c126f3da-a346-11e5-95a1-60812683554e.png)

Single site view.
![screen shot 2015-12-15 at 15 30 16](https://cloud.githubusercontent.com/assets/115071/11827404/e4280ece-a340-11e5-95af-93f851421124.png)

Mobile view
![screen shot 2015-12-15 at 16 25 03](https://cloud.githubusercontent.com/assets/115071/11828338/a513f83a-a348-11e5-8bb3-823050c801b4.png)

Mobile view:
![screen shot 2015-12-15 at 16 24 47](https://cloud.githubusercontent.com/assets/115071/11828343/a8940f7c-a348-11e5-99bb-d27fa05ac650.png)

Single Site view:
A plugin that doesn't have much info.
![screen shot 2015-12-15 at 16 13 01](https://cloud.githubusercontent.com/assets/115071/11828156/c457dfd8-a346-11e5-865f-8756aa4404b8.png)

Business Plugin:
![screen shot 2015-12-15 at 16 31 25](https://cloud.githubusercontent.com/assets/115071/11828425/7f678254-a349-11e5-8d3e-39082b33e443.png)


![screen shot 2015-12-15 at 16 08 51](https://cloud.githubusercontent.com/assets/115071/11828108/5de2ad64-a346-11e5-9242-c323f6d1f34b.png)




**To test**
Visit a single plugin page. For example 
http://calypso.localhost:3000/plugins/akismet and also http://calypso.localhost:3000/plugins/akismet/example.com

Browse plugins single plugin pages that have comptible versions with wp. 
Haven't been updated in a long time. 
Need an update.
Etc. 
Does it look and function as expected? 

cc: @rickybanister, @johnHackworth  and @lezama
